plugins/woocommerce/src/Admin/API/Reports/PerformanceIndicators/Controller.php

```diff
@@ -527,8 +527,14 @@ private function get_stats_parts( $full_stat ) {
 	 */
 	public function format_data_value( $data, $stat, $report, $chart, $query_args ) {
 		if ( 'jetpack/stats' === $report ) {
+			$index  = false;
+			$fields = $data['general']->visits->fields;
```

Comment:
We can still get three warnings from this line if `$data` isn't formatted correctly. Or worse, another fatal if `$data` is a string. Without looking at the context where this is used, it's marked as a `public` function and the description for the data field is `Data from external endpoint`. Based on that I wouldn't feel confident that it's always going to get passed with the correct format.